app.jsx

```diff
@@ -4,15 +4,20 @@ import { logo, center } from './data/ascii.js'
 import { user } from './lib/auth.js'
 import { RegistrationForm } from './component/registration_form.jsx'
 import { Header } from './component/header.jsx'
-import { Main, Footer, P, divider } from './component/elements.jsx'
+import { Main, Footer, P, divider, Link } from './component/elements.jsx'
+import { Router } from './lib/router.js'
 
 render(
   <>
-    <Header user={user} title="Akwaba NaN" />
-    <Main>
-      <P fg="comment">{logo}</P>
-      <RegistrationForm />
-    </Main>
+    <Header user={user} />
+    <Router>
+      <Main path="/profile">
+        <RegistrationForm />
+      </Main>
+      <Main path="*">
+        <P fg="comment">{logo}</P>
+      </Main>
+    </Router>
```

Comment:
This demonstrate how we can use the router component.
It acts as a big switch that will render elements if the `pathname` match the value of the `path` property of it's children